services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/ApplicationsResponse.kt

```diff
@@ -0,0 +1,3 @@
+package fi.hel.haitaton.hanke.application
+
+data class ApplicationsResponse(val applications: List<Application>)
```

Comment:
Could you have just returned the list of Applications from the controller?

Reply:
Maybe it is just my personal preference but I like to use a wrapper for lists. This enables extensions to the returned body without breaking changes to the api.

Reply:
Someone might say that you should not do stuff based on what _may_ be needed in the future. But yeah, prefer wrappers in these.